<a href="https://colab.research.google.com/github/TUIlmenauAMS/ClimateChangeCalculated/blob/main/climateChangeCalculated5.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# <center> Climate Change Calculated 5
## <center> How much space do we need for renewable energy?
### <center> Prof. Dr.-Ing. G. Schuller

### <center> Technical University Ilmenau

### <center> Institute for Media Technology



### This time we answer the question: How much space do we need?


In part 4 we answered the question: What can we do?
* There we saw that renewable energy is efficient and cheap enough to compete with and replace fossil energy.
* Now we answer the question: How much space will it need?


## Climate:
* We had an unusually warm winter in 2019/20, almost without snow and frost.
* During the Corona restrictions, the air became noticeably clearer.
* Less pollution, more sunshine.
* This shows the short-term effects of air pollution (and its removal)
* In winter 20/21 we had the following extremes: Extreme cold and snowfall, then unusual heat in February










* At the end of spring and summer 2021 there were already extreme temperatures in the northwest of the USA and in western Canada. Almost 50 C were reached there, in areas that usually have below 30C at the time!
* In Germany, high temperatures and permanent extreme heavy rains alternate with flood disasters as in western Germany.
* Record values are not only slightly exceeded globally, but equally significant! A characteristic of climate change.


* Pandemics such as Corona pandemic become more common as a result of climate change, because it shifts habitats and more often skips new viruses to new species.
* MunicRe Reinsurance calculates the rapidly increasing costs of weather-related natural disasters from 1980 to 2019:
https://www.munichre.com/de/risiken/extremwetter.html

* This indicates once again the urgency shown in my previous videos to reduce the global fossil CO2 emission to zero.


## What is the required area for renewable energy in Germany?
* Total electricity consumption from 2002 to 2019 (the pre-Corona year) and share of energy sources:
* [Fraunhofer ISE, Power Generation 2019](https://www.ise.fraunhofer.de/content/dam/ise/de/documents/news/2019/Stromerzeugung_2019_2.pdf)
* The graph on page 14 shows: From 2002 to 2019 only a small increase in electricity generation and consumption.
* In recent years, power consumption has even **decreased**!


* For an estimate of the future, we could therefore use the 2019 values.
* 2019: Total electricity generation (p. 13) **515 TWh**.
* Of which renewable energy sources : 237 TWh,
* 127 TWh wind and 47 TWh solar energy.






* However, there was recently (July 2021) an estimate of the electricity consumption in 2030 by the Federal Government:
[BMWI](https://www.bmwi.de/Redaktion/DE/Pressemitteilungen/2021/07/20210713-erste-abschaetzungen-stromverbrauch-2030.html)
* Result: "The mean of the forecast is **655 terawatt hours**. Among other things, 14 million electric cars, 6 million heat pumps and 30 terawatt hours of electricity for green hydrogen are included."
* In other words, by 2030 we need the difference in projected consumption to the renewable energy currently produced, 655 - 237 = **418 TWh** in addition.


**Note:**
* **1 kWh** (Kilo Watt Stunde) = 1000 Wh = 10³ Wh =1e3 Wh

* **1 TWh** (Terra Watt Stunde) = 10¹² Wh = 1e12 Wh



##Renewable energy yield risks
* Renewable energies are most reliable in distribution to **different energy sources, such as wind and sun**,
* also as seasonal compensation
* and geographical diversity, i.e. the use of the **European electricity grid**.
* See also: [Deutsche Wetter Dienst, Press Release 2018-03-06 Klima](https://www.dwd.de/DE/presse/pressemitteilungen/DE/2018/20180306_pressemitteilung_klima_pk_news.html)
* Therefore my **assumption:** *Photovoltaics and wind energy should take over 50% of each*, i.e. 418/2= **209 TWh/year**,


## Energy yield per square meter of solar cells:


In [ ]:
%%html
<iframe src="https://www.energie-experten.org/erneuerbare-energien/photovoltaik/planung/ertrag.html#c27405" width="900" height="900"></iframe>

* So global radiation in Germany: approx. **1000 kWh/sqm per year**
* Efficiency of modern solar cells: approx. 20%
* So energy per area per year: approx. **200 kWh/sqm per year**.


<!-- * The required area is calculated as: required energy per year/energy per area per year. -->
<!-- * We use "Python Sympy" for calculating units, also to verify the accuracy of the invoice: -->


## Area required solar cells for 209 TWh per year
Necessary Area = energy per year divided by energy per area per year =


In [ ]:
#@title  {display-mode: "form"}
import sympy
W=sympy.symbols("W") #unit "Watt"
h=sympy.symbols("h") #unit "hour"
yr=sympy.symbols("yr") #unit "Year"
m=sympy.symbols("m") #unit "meter"
EpAY= 200e3*W*h/(m**2)/yr #Energy per area per year
EpY=209e12*W*h/yr #desired Energy per year
print(" = ",EpY,"/",EpAY, " =", EpY/EpAY)
#print("Flaechenbedarf ist Energie pro Jahr geteilt durch Energie pro Fläche pro Jahr:") )

 =  209000000000000.0*W*h/yr / 200000.0*W*h/(m**2*yr)  = 1045000000.0*m**2


* Checking: The resulting unit (m2) is correct ("m**2" means m2 in Python)
* 106 m2 is 1 square kilometre (km2)
* This is 1045 km2.
* This area could be easily accommodated e.g. in the German above ground lignite mines:
* "The surface area of German above ground lignite mines in 2006 was more than **2300 km2**", sh.:


In [ ]:
%%html
<iframe src="https://de.wikipedia.org/wiki/Braunkohlebergbau" width="900" height="600"></iframe>

* The necessary solar cells could also be installed on **roofs**
* A great advantage of solar energy is its **decentrality**. Houses can be operated energy self-sufficiently.
* For solar cells suitable **roof surfaces** in Germany sum up to approx. 800 million square meters, according to
“Erneuerbare Energieträger in Deutschland: Potentiale und Kosten”
edited by Martin Kaltschmitt, Andreas Wiese, 1993, ISBN 978-3-642-93541-1.
* 800 million square metres are **800 square kilometres**, so the area is already almost the needed 1045 square kilometres.


## Needed Area for Wind Energy

* [Umweltbundesamt, Potential of wind energy](https://www.umweltbundesamt.de/sites/default/files/medien/378/publikationen/potenzial_der_windenergie.pdf)
* Chapter 4.1, p.16: ... a radial distance of the 4-fold rotor diameter...
* Hence the radial wind turbine distance is **456 m**.
* For a square arrangement, this corresponds to an area of **456² square metres =207936 square metres**
* Page 15: Power per wind turbine: **3.2 MW = 3.2e6 W**
* In the summary, page 2: "... in
National average **2440 full hours per year**..."
<!-- * **Result:** Energy per year per sqm = **37.55 kWh/sqm per year** -->




## Wind energy per square meter per year
Energy per square meter per year = power times full hours per year divided by area =


In [ ]:
#@title  {display-mode: "form"}
import sympy
m=sympy.symbols("m") #unit "meter"
W=sympy.symbols("W") #unit "Watt"
h=sympy.symbols("h") #unit "hour"
yr=sympy.symbols("yr") #unit "Year"
P=3.2e6*W #Power of win turbine
fr=2440*h/yr # Hours per year of this power (fraction)
A=456**2*m**2 #Area covered by wind turbine
EpAYwind=P*fr/A
print(P,"*", fr, "/", A," = ", EpAYwind)
#print("Die Energie in Wh geteilt durch die Stunden im Jahr ergibt die Leistung, in Watt pro qm ")

3200000.0*W * 2440*h/yr / 207936*m**2  =  37550.0153893506*W*h/(m**2*yr)


Also 37550 Wh pro Jahr pro qm.

## Necessary area for 209 TWh of wind per year:
Necessary area = energy needed per year divided by energy per square meter =


In [ ]:
#@title {display-mode: "form"} 
print(" = ", EpY, "/", EpAYwind, " = ",  EpY/ EpAYwind)

 =  209000000000000.0*W*h/yr / 37550.0153893506*W*h/(m**2*yr)  =  5565909836.06557*m**2


* This is about 5566 square kilometers.
* The total area of Germany is 357386 sq km, the additional area necessary for wind power is therefore about 1.6% of it.
* About half of the area of Germany is arable land, i.e. it would be additionally only about 3% of the arable land.


## Conclusions
* The required renewable energy for the projected demand in 2030 can be produced **practicable in the country**.
* For the area of the solar cells, decommissioned **above ground lignite mines** would be more than sufficient.
* The **suitable roof areas** would be almost completely sufficient
* For wind power, an additional **3% of the arable area** would be needed, which is still available for agriculture.


* Electricity storage is needed to balance the fluctuations of wind and sun
* More about this in a following episode
